includes/cenchat-meta.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class Cenchat_Meta {
+    public function add_meta() {
+        $options = get_option( 'cenchat_id' );
+
+        require_once dirname( dirname( __FILE__ ) ) . '/public/partials/meta.php';
```

Comment:
Use [`plugin_dir_path`](https://developer.wordpress.org/reference/functions/plugin_dir_path/) instead of nested `dirname()`.